### DISCUSSION TITLE
Onclicking `Robolectric plugin for Chrome` showing error

### DISCUSSION THREAD

**Discussion by hellosagar** [robolectric/robolectric#7108] [Q&A] [answered]
Seems dumb to raise but when I clicked on `Robolectric plugin for Chrome` in this [page](http://robolectric.org/writing-a-test/), its showing error on this [link](http://robolectric.org/writing-a-test/)

**Comment by utzcoz** [comment]
Sorry for that. Chrome extension is not accessible now. I have created a discussion to make it alive again: https://github.com/robolectric/robolectric/discussions/7102.

**Comment by utzcoz** [comment, marked as answer]
We will decide to drop chrome extension of Robolectric. See the discussion https://github.com/robolectric/robolectric/discussions/7102.
